x/ibc/core/02-client/keeper/proposal.go

```diff
@@ -70,3 +70,35 @@ func (k Keeper) ClientUpdateProposal(ctx sdk.Context, p *types.ClientUpdatePropo
 
 	return nil
 }
+
+// UpgradeProposal sets the upgraded client state in the upgrade store. It clears
+// an IBC client state and consensus state if a previous plan was set. Then  it
+// will schedule an upgrade and finally set the upgraded client state in upgrade
+// store.
+func (k Keeper) UpgradeProposal(ctx sdk.Context, p *types.UpgradeProposal) error {
```

Comment:
I prefer this is called `DoIBCUpgrade` or something similar

Reply:
`HandleIBCUpgrade`?

Reply:
Should I update the function `ClientUpdateProposal` as well?